readme.md

```diff
@@ -1,4 +1,13 @@
 # Marvin
+
 The new Wikipedia mobile website. Read more on our [wiki]!
 
-[wiki]: https://www.mediawiki.org/wiki/Reading/Web/Projects/NewMobileWebsite
\ No newline at end of file
+## Development
+
+Marvin requires the Node.js active LTS version, and npm > 5.X.
```

Comment:
Great idea. Addressed in 29f3824, also the npm version.

I've kept some info in the README because it is a bit more friendly than just linking people away from the documentation into a README file.